src/api/modules/utils.ts

```diff
@@ -75,15 +79,21 @@ export const validateAppAction = async (operation: string, app?): Promise<boolea
   const workspaces = createWorkspacesClient()
   const workspaceMeta = await workspaces.get(account, workspace)
   if (workspaceMeta.production && !contains(operation, workspaceProductionAllowedOperatios)) {
-    throw new CommandError(workspaceProductionMessage(workspace))
+    throw ErrorReport.createAndMaybeRegisterOnTelemetry({
+      kind: ErrorKinds.FLOW_ISSUE_ERROR,
+      originalError: new Error(workspaceProductionMessage(workspace)),
+    })
   }
 
   // No app arguments and no manifest file.
   const isReadable = await ManifestEditor.isManifestReadable()
   if (!app && !isReadable) {
-    throw new CommandError(
-      `No app was found, please fix your manifest.json${app ? ' or use <vendor>.<name>[@<version>]' : ''}`
-    )
+    throw ErrorReport.createAndMaybeRegisterOnTelemetry({
+      kind: ErrorKinds.FLOW_ISSUE_ERROR,
```

Comment:
Maybe FlowIssues are so common that we would want a utility for create them?
You could create in error/utils.ts something like:
```ts
/** Create and register on telemetry an ErrorReport with errorKind = ErrorKinds.FLOW_ISSUE_ERROR */
export const createFlowIssueError(msg: string) {
  return ...
}
```